src/plinth/browser/element.gleam

```diff
@@ -170,3 +171,6 @@ pub fn get_checked(element: Element) -> Bool
 
 @external(javascript, "../../element_ffi.mjs", "contains")
 pub fn contains(element: Element, other: Element) -> Bool
+
+@external(javascript, "../../element_ffi.mjs", "getClassList")
```

Comment:
I think most of the other api's in plinth are without the get. both in Gleam and the JS helper